eland/field_mappings.py

```diff
@@ -479,7 +477,7 @@ def _pd_dtype_to_es_dtype(pd_dtype):
         category NA NA Finite list of text values
         ```
         """
-        es_dtype = None
+        es_dtype: str = None
```

Comment:
This function can return `None` so should be `Optional[str]` both here and return type?

Reply:
Yep, Missed in plain sight 👍